product_pricelist_supplierinfo/models/product_template.py

```diff
@@ -9,8 +9,8 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    def _get_supplierinfo_pricelist_price(
-            self, rule, date=None, quantity=None, product_id=None):
+    def _get_supplierinfo_domain(
+            self, rule, date=None, quantity=None, product_id=None, partner_id=False):
```

Comment:
Where are you using the new parameter `partner_id`?

Reply:
Hi @moylop260 After the merge of the PR 668, I changed the way I was using the new parameter, and now I'm using it on the method `_select_seller`